### ISSUE TITLE
Quotes are not being properly escaped

### ISSUE THREAD

**Issue by sbidin** [horejsek/python-fastjsonschema#40]
The following schema will generate invalid code:

``` json
{
    "enum": ["Bernardo O'Higgins"]
}
```

This is the result:

``` python
if data not in ["Bernardo O'Higgins"]:
    raise JsonSchemaException("data must be one of ["Bernardo O'Higgins"]")
```

Note that quotes are not properly escaped, resulting in invalid Python code.

**Comment by sbidin** [comment]
For those in a similar bind, I'm using the following instead of `fastjsonschema.compile` to patch over this issue:

``` python
import re
import fastjsonschema

def compile_fastjsonschema_hack(schema, handlers):
    resolver, codegen = fastjsonschema._factory(schema, handlers)+
    state = codegen.global_state
    code = codegen.func_code
    code = re.sub(r'must be one of.+$', 'invalid")', code, flags=re.MULTILINE)
    exec(code, state)
    return state[resolver.get_scope_name()]
```

**Comment by horejsek** [comment]
Fixed in version 2.8, thanks for the report. :-)
